brave-core/src/main/java/com/github/kristofa/brave/AnnotationSubmitter.java

```diff
@@ -191,9 +186,9 @@ public void submitBinaryAnnotation(String key, int value) {
         submitBinaryAnnotation(key, String.valueOf(value));
     }
 
-    long currentTimeMicroseconds(@Nullable Long startTick) {
-        return startTick != null
-            ? (System.nanoTime() - startTick) / 1000
```

Comment:
Should this fix (or bug fixes in general) be ported back to previous versions of the library as a patch version (e.g. v3.13.0 -> v3.13.1)? Or is the expectation that users should just bump to the latest minor + patch (in this case v3.14.1)?